resources/services/elasticsearch/elasticsearch_domains.go

```diff
@@ -18,7 +18,7 @@ func ElasticsearchDomains() *schema.Table {
 		Multiplex:    client.ServiceAccountRegionMultiplexer("es"),
 		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
 		DeleteFilter: client.DeleteAccountRegionFilter,
-		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "id"}},
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "id", "region"}},
```

Comment:
```suggestion
		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "region", "id"}},
```

This is a better PK and more inline with the rest of region defined pks, would require a change in the SQL.